applications/StructuralMechanicsApplication/custom_processes/prism_neighbours_process.h

```diff
@@ -0,0 +1,306 @@
+// KRATOS  ___|  |                   |                   |
```

Comment:
is this file specific to the StructuralMechanicsApp?

Reply:
yes, do you want me to move it to the core?